x/mock/test_utils.go

```diff
@@ -1,6 +1,8 @@
 package mock
 
 import (
+	"fmt"
+	"github.com/cosmos/cosmos-sdk/x/auth"
```

Comment:
Igor told me that we can use following script to format the imports: 

```
go get -u github.com/AanZee/goimportssort

for entry in `find . -name "*.go"`; do
    echo $entry
    if grep -q "DO NOT EDIT" "$entry"; then
      echo "xxxxxxxx=================================="
      continue
    fi
    goimportssort -w -local github.com/bnb-chain/ $entry
done
```

Reply:
added